website/docs/docs/collaborate/manage-access/set-up-snowflake-oauth.md

```diff
@@ -41,7 +41,7 @@ CREATE OR REPLACE SECURITY INTEGRATION DBT_CLOUD
 | ENABLED  | Required |
 | OAUTH_CLIENT  | Required |
 | OAUTH_CLIENT_TYPE  | Required |
-| OAUTH_REDIRECT_URI  | Required. If dbt Cloud is deployed on-premises, use the domain name of your application instead of `cloud.getdbt.com` |
+| OAUTH_REDIRECT_URI  | Required. If dbt Cloud is deployed on-premises, use the domain name of your application instead of `cloud.getdbt.com`. If you're on the EMEA servers, use `emea.dbt.com` for the redirect URI. |
```

Comment:
I wonder if we should reword slightly and link to this [regions doc](https://docs.getdbt.com/docs/deploy/regions) so when we add APAC, we don't need to edit this page again?

Reply:
i totally agree with Leona! i linked to the regions doc in our [database page](https://docs.getdbt.com/docs/get-started/connect-your-database) anticipating the apac instance change.